### ISSUE TITLE
web ui settings

### ISSUE THREAD

**Issue by zivio** [asapach/peerflix-server#103]
Can I add a web ui settings "connections" and "verify" that would be applicable during peerflix-server. This would help to work on slow devices.

Можно ли добавить в web ui настройку "connections" и "verify" которая бы применялась во время работы peerflix-server. Это бы помогло для работы на медленных устройствах.

**Comment by asapach** [comment]
I don't think you can change the connection settings in [torrent-stream](https://github.com/mafintosh/torrent-stream) after it has been instantiated and I haven't tried turning off the `verify` option as you have noticed in #100, so I don't really see the point.
